oak/server/rust/oak_runtime/src/lib.rs

```diff
@@ -421,6 +432,26 @@ impl Runtime {
     pub fn gather_metrics(&self) -> Vec<MetricFamily> {
         self.metrics_data.gather()
     }
+
+    /// Check Wasm module signatures.
+    pub(crate) fn verify_module_signatures(&self) -> Result<(), OakStatus> {
+        for (name, module) in &self.application_configuration.wasm_modules {
+            // Get signature by module name.
+            if let Some(signature_item) = self.signature_configuration.get(name) {
```

Comment:
Looks like having a signature is optional. Maybe add a comment somewhere explaining what that means?